products/llm_analytics/frontend/llmAnalyticsLogic.test.ts

```diff
@@ -0,0 +1,146 @@
+import { initKeaTests } from '~/test/init'
+import { ChartDisplayType } from '~/types'
+
+import { llmAnalyticsLogic } from './llmAnalyticsLogic'
+
+describe('llmAnalyticsLogic', () => {
+    let logic: ReturnType<typeof llmAnalyticsLogic.build>
+
+    beforeEach(() => {
+        initKeaTests()
+        logic = llmAnalyticsLogic()
+        logic.mount()
+    })
+
+    afterEach(() => {
+        logic.unmount()
+    })
+
+    describe('tiles configuration', () => {
+        it('should have explicitDate set to true for aggregate display tiles', () => {
+            const tiles = logic.values.tiles
+
+            // Find the aggregate display tiles
+            const totalCostTile = tiles.find((t) => t.title === 'Total cost (USD)')
+            const costByModelTile = tiles.find((t) => t.title === 'Cost by model (USD)')
+            const generationsByStatusTile = tiles.find((t) => t.title === 'Generations by HTTP status')
+
+            // Verify Total cost tile
+            expect(totalCostTile).not.toBeUndefined()
+            expect(totalCostTile?.query.dateRange?.explicitDate).toBe(true)
+            expect(totalCostTile?.query.trendsFilter?.display).toBe(ChartDisplayType.BoldNumber)
+
+            // Verify Cost by model tile
+            expect(costByModelTile).not.toBeUndefined()
+            expect(costByModelTile?.query.dateRange?.explicitDate).toBe(true)
+            expect(costByModelTile?.query.trendsFilter?.display).toBe(ChartDisplayType.ActionsBarValue)
+
+            // Verify Generations by HTTP status tile
+            expect(generationsByStatusTile).not.toBeUndefined()
+            expect(generationsByStatusTile?.query.dateRange?.explicitDate).toBe(true)
+            expect(generationsByStatusTile?.query.trendsFilter?.display).toBe(ChartDisplayType.ActionsBarValue)
+        })
+
+        it('should NOT have explicitDate set for time-series display tiles', () => {
```

Comment:
These tests could actually be nicely generalized, because right now they'll break if e.g. "Generations by HTTP status" is changed to something else.
Instead, they could filter 1. all insights where display is in `NON_TIME_SERIES_DISPLAY_TYPES` (aggregate ones), 2. all insights where display is NOT in `NON_TIME_SERIES_DISPLAY_TYPES` (time-series ones). More resilient this way.